01_buildconfig

```diff
@@ -1,5 +1,7 @@
 APT::Get::Assume-Yes "true";
-APT::Get::force-yes "true";
+APT::Get::allow-downgrades "true";
+APT::Get::allow-remove-essential "true";
+APT::Get::allow-change-held-packages "true";
```

Comment:
Have you verified that these 3 options are available in the oldest flavour we support (i.e wheezy)?

Reply:
@petrosagg They're not available, `force-yes` is replaced by these 3 options since apt v1.1 (https://manpages.debian.org/stretch/apt/apt-get.8.en.html). However, after carefully reading about these options, I think we shouldn't include them in our base images since they can potentially cause unexpected packages installed or removed. Also official Debian images don't have this option included so I think we should remove these options.

Created a PR to remove them in debian images here https://github.com/resin-io-library/base-images/pull/367

Reply:
@nghiant2710 I didn't understand your answer. I know `force-yes` is not available in the new versions of debian and that it was replaced by some other options. My question is if those other options are available in the old versions of debian since you seem to replace force-yes completely.

I don't think we should revert this behaviour. User builds will break. Debian guidelines have desktop and servers in mind, not containers and Dockerfiles

Reply:
@petrosagg I meant the new options are not available in the old versions and reading about new options make me a bit worry since they can cause unexpected packages installed or removed.

However, since you're fine with keeping it, I've updated the PR to set the correct options for each Debian version.
Please review the PR that connects to this one as well https://github.com/resin-io-library/base-images/pull/367

Reply:
I just realised that we set `Assume-yes` which the same as passing `-y` in the apt-get install command line which is the feature I wanted our based images to have. Given this new info I agree that we should remove force yes and the other new options and leave only Assume-yes